### ISSUE TITLE
Save timing information in renderdoc capture

### ISSUE THREAD

**Issue by dev-joss** [baldurk/renderdoc#2325]
## Description

Ability to capture graphics API timings from system that capture was created on, and save in render doc file for offline analysis.

Currently the timing functionality of render executes the serialized draw call on the computer running renderdoc. Its not possible to save this information into the renderdoc file in order to analyze on another system.

Similarly, timing information is not saved for the frame that was captured, but only after replaying the draw calls via renderdoc.

The situation where this would be needed is when profiling application performance on a third parties system. Ideally the third party would create a renderdoc capture, via renderdoc, or the renderdoc API integrated with the application.

The third party would send the renderdoc file and a developer could use it as basis for optimization.

## Environment

* RenderDoc version: 1.14
* Operating System: Windows 10
* Graphics API: DirectX 11

**Comment by baldurk** [comment]
RenderDoc is not a profiler so this kind of feature is not in scope for the project. The hardware counters obtained can be exported to CSV, or you could use the remote replay feature if you have access to the machine that you want to run the replay on, but that's the only thing.
